pkg/servicecentral/it/BUILD.bazel

```
@@ -0,0 +1,23 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_test")
+
+go_test(
+    name = "go_default_test",
+    testonly = False,
```

```suggestion
    size = "medium",
```
Don't need `testonly` (in the other place it allows `pkg_tar` rule to depend on the e2e test binary so that it can package it into a tar) but need `size` (tells bazel how "big" is the test. Mostly to avoid warnings).